.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Ran ESLint formatter(--fix)
+7fa3b3bb0ed3e493ab4a6dd0a5ff13e185417f02
```

Comment:
What is this?

Reply:
The commit where the formatter runs on the project. https://github.com/open-sauced/browser-extensions/pull/49/commits/7fa3b3bb0ed3e493ab4a6dd0a5ff13e185417f02. Making a lot of formatting changes. @diivi suggested adding it to this file to hide the loads of formatting changes when viewing code in blame.

Reply:
I read about it here, https://github.blog/changelog/2022-03-24-ignore-commits-in-the-blame-view-beta/.